Sections/US-States/FL/GPP Extension: IAB Privacy’s Florida Privacy Technical Specification.md

```diff
@@ -0,0 +1,190 @@
+<h1 id="gpp-extension-iab-privacy-s-florida-privacy-technical-specification">GPP Extension: IAB Privacy’s Florida Privacy Technical Specification</h1>
+<h2 id="about-this-document">About this document</h2>
+<p>The global standard <a href="https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform">GPP</a> defines a way for local standards to &quot;plug-in&quot; into the existing mechanics defined by GPP and the <a href="https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform/blob/main/Core/CMP%20API%20Specification.md">GPP client side API</a>. This document outlines the technical specification for using the GPP specifications with the IAB Privacy Multi-State Privacy Agreement legal requirements.</p>
+
+<h3>Version History&nbsp;</h3>
+<div>
+<table>
+<tbody>
+<tr>
+<td><strong>Date</strong></td>
+<td><strong>Version</strong></td>
+<td><strong>Comments</strong></td>
+</tr>
+
+<tr>
+<td>May 2024</td>
+<td>1.0</td>
+<td>Version 1.0 released</td>
+</tr>
+</tbody>
+</table>
+</div>
+
+<h2>Florida Section</h2>
+<p>The Florida Privacy String consists of the following components. Users should employ the Florida Privacy String only if they have determined the Florida Digital Bill of Rights, Fla. Stat. § 501.701 et seq., applies to their processing of a consumer’s personal data.</p>
```

Comment:
Why can't people use it if they are either not sure or if they want to voluntarily comply with the requirements even when they don't apply to the company.

Reply:
During the working group meetings we voted to keep the same structure for the new states to help accelerate the development of the sections. This language is consistent with the language in the sections for California, Colorado, Connecticut, Utah, and Virginia. I’ve attached a screenshot of the language from each of those sections. Similar updates also exist for MT, OR, and TX to keep everything consistent.
![Screenshot 2024-06-05 at 3 07 14 PM](https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform/assets/126587246/57a5093f-e85b-4362-824b-5fd9f967229c)

Reply:
the consistency is the problem, in every other jurisdiction, residency or location determines applicability, but in florida, publishers have other criteria to consider and may not even know if the law applies

Reply:
Reflecting the slack discussion, deleting the word only, changing 'applies' to 'may apply', or both, could solve.

Reply:
proposed update:
"Users should employ the Florida Privacy String if they have determined the Florida Digital Bill of Rights, Fla. Stat. § 501.701 et seq., applies to their processing of a consumer's personal data."